content/1. Leertaken/WEBDEV/Security/Workshops/10. Workshop Git/2. Git Repositories.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 2. Git Repositories
+taxonomie: None
+tags: 
+draft: true
+---
+
+> [!warning] Al een beetje bekend met Git? Dan kun je het eerste deel van deze workshop overslaan en doorgaan vanaf [[3. Git Forks & Branches]]
+
+## Opdracht 1: Repository aanmaken
+Een repository is de opslagplaats van je project. Hierin staan alle bestanden en de volledige geschiedenis van hun wijzigingen.
+
+**Stap 1:** Log in op GitHub en ga naar de homepagina of je dashboard.
+
+**Stap 2:** Klik op de groene ‘New’ knop aan de linkerkant van de pagina.
+
+> [!info] Houd de naam van je repository simpel en beschrijvend. Vermijd spaties en speciale tekens (behalve -).
+
+**Stap 2.1:** Geef de repository een naam (bijvoorbeeld FirstRepository) en een beschrijving. Kies ook of je de repository public of private wilt maken.
+
+**Stap 2.2:** Voeg een .gitignore bestand toe met het template voor VisualStudio.
+
+**_//toevoegen aan ondersteunende-informatie en verwijzing toevoegen_**
+> [!info] Een .gitignore is een bestand dat aangeeft welke bestanden of mappen Git moet negeren. Voor een .NET-project kun je het Visual Studio-template kiezen om bijvoorbeeld tijdelijke en build-bestanden uit te sluiten.
+
+**Stap 2.3:** Klik op de Create Repository knop.
+
+Je hebt nu je eerste repository aangemaakt. Gefeliciteerd!
+
+  
+## Opdracht 2: Repository clonen
+Nu je een repository hebt aangemaakt, kunnen we deze clonen naar je PC of laptop. In deze workshop gebruiken we GitHub Desktop. Dit programma maakt het beheren van repositories eenvoudig.
+
+**Stap 1:** Open GitHub Desktop en log in met je GitHub account.
+	_Dit moet hetzelfde account zijn waarmee je net de repository hebt aangemaakt._
+
+**Stap 2:** Klik linksboven op de knop Current Repository.
+
+**Stap 3:** Klik op Add en selecteer vervolgens Clone Repository.
+
+**Stap 3.1:** Blijf onder het kopje GitHub.com en zoek onder Your Repositories naar de repository die je zojuist hebt aangemaakt.
+
+> [!info] **TIP:** Bewaar al je repositories in één folder op je PC/laptop.
```

Comment:
Denk het op zich niet, ik heb zelf ook repos op verschillende plekken staan voor student-assistent of een KBS project, zou deze eruit halen